src/components/Kanban/index.tsx

```diff
@@ -0,0 +1,110 @@
+import { DragDropContext, Droppable } from "react-beautiful-dnd";
+import Table from "../Table";
+import LogBar from "../LogBar";
+import { useSelector } from "react-redux";
+import { RootState } from "../../store";
+import { addButton, card, messageWrapper, wrapperKanban, wrapperTables } from "./style.css";
+import { useState } from "react";
+import AddInput from "../AddInput";
+import {
+  addTable,
+  addLog,
+  changeTableOrder,
+  changeTaskOrder,
+} from "../../store/newBoardSlice";
+import { useDispatch } from "react-redux";
+
+function Kanban() {
+  const [addTableVar, setAddTable] = useState(false);
+  const board = useSelector((state: RootState) => state.newBoardSlice.selected);
+  const tables = useSelector(
+    (state: RootState) => state.newBoardSlice.selected?.tables
+  );
+  const dispatch = useDispatch();
+
+  function handleOnDragEnd(result: any) {
+    if (result.type === "COLUMN") {
+      dispatch(
+        changeTableOrder({
+          from: +result.source.index,
+          to: +result.destination.index,
```

Comment:
Please avoid increment/decrement operators when possible, it's really difficult to understand if you want to mutate something or just get value + 1 
The only use case where these operators are fine is `for` loop